contracts/sweep/SweepERC20.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./TokenSweep.sol";
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+
+/**
+ * @title Adds the ability to sweep ERC20 tokens to a beneficiary address
+ */
+abstract contract SweepERC20 is TokenSweep {
+    /**
+     * @notice Sweep the erc20 tokens to the beneficiary address
+     **/
+    function _sweepERC20Tokens(address token, uint256 amount) internal {
+        require(address(token) != address(this), "SweepERC20: self-transfer");
```

Comment:
question: If `token` is type address, does it still need casting to `address`?

Suspect you can drop the casting of `token` in both the requires

Reply:
relic from when the token param was IERC20!